com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/AttachRequestHandler.java

```diff
@@ -58,6 +59,18 @@ public void handle(Command command, Arguments arguments, Response response, IDeb
                     attachArguments.timeout);
             context.setDebugSession(debugSession);
             logger.info("Attaching to debuggee VM succeeded.");
+
+            // If the debugger and debuggee run at the different JVM platforms, show a warning message.
+            if (debugSession != null) {
+                String debuggeeVersion = debugSession.getVM().version();
+                String debuggerVersion = System.getProperty("java.version");
+                if (!debuggerVersion.equals(debuggeeVersion)) {
+                    String warnMessage = String.format("[Warn] Remote debuggee JVM: %s, but debugger JVM: %s. The source mapping for jre system libraries "
```

Comment:
The debugger and the debuggee are running in different versions of JVMs. You could see wrong source mapping results.
Debugger JVM version: ***
Debuggee JVM  version: ***

Reply:
fixed.